README.md

```diff
@@ -107,6 +108,8 @@ alt="platformatic"
 * `noStyle` - *`boolean`* - if set, no style rules will be added. Especially useful when combined with [Prettier](https://prettier.io/), [dprint](https://dprint.dev/) or similar
 * `semi` - *`boolean`* - if set, enforce rather than forbid semicolons (same as `semistandard` did)
 * `ts` - *`boolean`* - if set, TypeScript syntax will be supported and `*.ts` (including `*.d.ts`) will be checked. To add additional file patterns to the TypeScript checks, use `filesTs`
+* `react` - *`'recommended' | 'all'`* - selected rule set from `eslint-plugin-react`, see [JSX support](#jsx-support) for details
+* `requireReactImport` - *`boolean`* - if set, import of React is required whenever JSX is used, see [JSX support](#jsx-support) for details
```

Comment:
Why that option is added?

Reply:
Old React 16 and older require `import React from 'react'` to be in the file if there is any JSX in the file. While React 17+ no longer needs that. 
ESLint, even with `neostandard` rules would scream that you are importing React const, but you never use it (because you do not have `React.*` in the code, just JSX). React plugin has the rule (in 'recommended' and 'all' rulesets) that overrides `no-unused-vars`. So you do not get these false positive errors and linter complaining about importing React but not using it. Because React16 and older would not work, if React is not imported, and their plugin tells you that you have to import React (in case it is not imported).
When you use React 17+, you should also extend from `jsx-runtime` (not only `recommended`/`all`) to disable their rules and revert the logic (to tell you that you are importing React, but you do not have to).

Now... because I do not know how to get React version in the repo, where neostandard is used, I have added this switch with reverted logic (because I expect majority already use React17+). Those who have repo with old React has to enable `requireReactImport`. Then `jsx-runtime` is not used.

Reply:
Please use the '16-' or '17+' (the strings expected in `version`) as `react` tunable value instead of shareable configs.